recipe/meta.yaml

```
@@ -19,40 +19,41 @@ build:
 
 requirements:
   host:
-    - python >=3.6.1,<4.0
+    - python >=3.7.2,<4.0
     - pip
     - poetry-core >=1.0.0
 
   run:
     - backports.zoneinfo  # https://github.com/conda-forge/django-feedstock/issues/134
     - dodgy >=0.2.1,<0.3.0
-    - mccabe >=0.6.0,<0.7.0
+    - flake8 <6.0.0
+    - gitpython >=3.1.27
```

Could be misunderstanding, but should this be?

```suggestion
    - gitpython >=3.1.27,<4
```

Looking at [this upstream requirement]( https://github.com/PyCQA/prospector/blob/8754387c6cf746b812f75994025bf3a98078239a/pyproject.toml#L62 )